webviews/src/views/RokuAutomationView/ConfigControls.svelte

```diff
@@ -0,0 +1,141 @@
+<script>
+    export let selectedConfig; // string
+    export let configs; // array of configs
+
+    let showModal;
+    let inputValue = '';
+    let _inputValue = inputValue;
```

Comment:
What's the reason for having both `inputValue` and `_inputValue`?

Reply:
_inputValue changed at a high cadence (with every key press) and needed inputValue to update on commit - may have changed since I added and will revisit this